src/furtherWork/petG.js

```diff
@@ -0,0 +1,70 @@
+// GAME STYLE PET
+const {
+    grootTypes,
+    battleCries,
+    attackMoves
+} = require("./gameProperties.js");
+
+
+POTION_HEAL = 10;
+
+
+class Groot {
+    constructor(name, moves=[]) {
+        this.name = name
+        this.battlecry = battleCries[Math.floor(Math.random() * battleCries.length)];
+        this.moves = moves;
+        this.species = grootTypes[Math.floor(Math.random() * grootTypes.length)];
+        this.weakness = this.weakness;
+        this.health = 100;
+       
+    }
+
+get isAlive() {
```

Comment:
Everything from here to line 50 should be indented a layer as it is inside the `Groot` class